argilla/src/argilla/datasets/_export/_disk.py

```diff
@@ -12,38 +12,40 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-from abc import ABC
 import json
 import logging
 import os
-from pathlib import Path
 import warnings
-from typing import Optional, Union, TYPE_CHECKING, Tuple, Type
+from abc import ABC
+from pathlib import Path
+from typing import TYPE_CHECKING, Optional, Tuple, Type, Union
 from uuid import uuid4
 
 from argilla._models import DatasetModel
 from argilla.client import Argilla
 from argilla.settings import Settings
 from argilla.workspaces._resource import Workspace
 
-
 if TYPE_CHECKING:
     from argilla import Dataset
 
 
 class DiskImportExportMixin(ABC):
     """A mixin for exporting and importing datasets to and from disk."""
 
-    _model: DatasetModel
-    _default_settings_path = "settings.json"
-    _default_records_path = "records.json"
-    _default_dataset_path = "dataset.json"
+    _MODEL: DatasetModel
```

Comment:
I think this is not a constant since no value has been assigned.

```suggestion
    _model: DatasetModel
```